src/main/cpp/hw/factories/DigitalInputFactory.cpp

```diff
@@ -22,7 +22,7 @@
 // Team 302 includes
 #include "hw/DragonDigitalInput.h"
 #include "hw/factories/DigitalInputFactory.h"
-#include "configs/usages/DigitalInputUsage.h"
+#include "configs/RobotElementNames.h"
```

Comment:
Shouldn't this factory go away?  Isn't the code generator the thing that is create DigitalInputs now?

Reply:
Added a story